adminer/drivers/elastic.inc.php

```diff
@@ -8,64 +8,75 @@
 		class Min_DB {
 			var $extension = "JSON", $server_info, $errno, $error, $_url, $_db;
 
-			/** Performs query
-			 * @param string
-			 * @param array
-			 * @param string
-			 * @return mixed
+			/**
+			 * @param string $path
+			 * @param array|null $content
+			 * @param string $method
+			 * @return array|false
 			 */
-			function rootQuery($path, $content = array(), $method = 'GET') {
+			function rootQuery($path, array $content = null, $method = 'GET') {
 				@ini_set('track_errors', 1); // @ - may be disabled
 				$file = @file_get_contents("$this->_url/" . ltrim($path, '/'), false, stream_context_create(array('http' => array(
 					'method' => $method,
 					'content' => $content === null ? $content : json_encode($content),
 					'header' => 'Content-Type: application/json',
 					'ignore_errors' => 1, // available since PHP 5.2.10
```

Comment:
```suggestion
					'ignore_errors' => 1, // available since PHP 5.2.10
					'follow_location' => 0,
					'max_redirects' => 0,
```
Prevent redirection? First line is unchanged but GitHub suggestion feature doesn't seem to allow insert-only changes.

Reply:
Thanks, I did it in all HTTP based drivers.